### ISSUE TITLE
List of all dropdowns[Arabic]

### ISSUE THREAD

**Issue by talhakpg** [talhakpg/TeleTashafi#35]

**Comment by hassanahak** [comment]
Please provide all drop dawn list with Arabic to Talha.

**Comment by hassanahak** [comment]
Yes Mahinder, the QA tester are testing systematically. Most likely will not be able to collect all the drop down list . If you kindly provide me with all drop down list, I will translate them.
